WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
Fair enough.

Then I'll change my stance: Armenians should be able to create their culture's Empire at-will, regardless of religion.